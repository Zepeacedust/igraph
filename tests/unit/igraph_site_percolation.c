/*
    IGraph library.
    Copyright (C) 2025  The igraph development team <igraph@igraph.org>

    This program is free software; you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation; either version 2 of the License, or
    (at your option) any later version.
 
    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.
 
    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <https://www.gnu.org/licenses/>.
*/

#include "test_utilities.h"
#include <igraph.h>

int main(void) {
  // test igraph_i_edge_list_percolation
  
  
  // test igraph_bond_percolation
  
  
  // test igraph_site_percolation

  VERIFY_FINALLY_STACK();
  return 0;
}
