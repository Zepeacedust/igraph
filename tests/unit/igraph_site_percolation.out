K_5 graph percolation curve, no provided vertex sequence:
( 1 2 3 4 5 )
C_4 graph with vertex sequece 0, 2
( 1 1 3 4 )
Karate graph biggest component size, no edge list given: 34
Generated disconnected graph, 100 vertices, p=0.01
Sizes match :)
